cw_monero/lib/monero_unspent.dart

```diff
@@ -7,27 +7,29 @@ class MoneroUnspent extends Unspent {
   MoneroUnspent(
       String address, String hash, String keyImage, int value, bool isFrozen, this.isUnlocked)
       : super(address, hash, value, 0, keyImage) {
+    getCoinByKeyImage(keyImage).then((coinId) {
+      if (coinId == null) throw Exception("Unable to find a coin for address $address");
```

Comment:
is this exception gracefully handled? or would it cause a gray screen or so?

Reply:
the behavior is more or less the same, except it would throw it in constructor instead of later in getter.

Reply:
ah no wait